### DISCUSSION TITLE
Does Stylelint support?

### DISCUSSION THREAD

**Discussion by dukexie** [web-infra-dev/rsbuild#3591] [Q&A] [answered]
搜了很多地方，发现没有 [Stylelint](https://stylelint.io/) 相关的说明。

近期把公司的项目从 Webpack + Vite 迁移到 Rsbuild 上，目前对接到 Sylelint 时发现找不到代替的模块。

**Comment by chenjiahan** [comment, marked as answer]
You can follow Stylelint's official documentation to set it up.

If you need to run stylelint during compilation, try this plugin: https://github.com/webpack-contrib/stylelint-webpack-plugin
